app/models/changelog_data.php

```diff
@@ -0,0 +1,1468 @@
+            'section' => ['end_user'],
+            'type'    => ['better'],
+            'authors' => ['pascal'],
+        ],
+        [
+            'message' => ['Data storage no longer aggressively escape HTML tags, quotes and special characters such as &amp; and /, which makes searching for HTML tags or things like &amp;BrandShortName; or \n possible now'],
+            'section' => ['end_user'],
+            'type'    => ['better'],
+            'authors' => ['pascal'],
+        ],
+        [
+            'message' => ['Faster API response (3x on average) by serializing our datasets with PHP before using them'],
+            'section' => ['external_api'],
+            'type'    => ['better'],
+            'issues'  => ['303'],
+            'authors' => ['pascal'],
+        ],
+        [
+            'message' => ['Fixed various API regressions and bugs reported by Keko'],
+            'section' => ['external_api'],
+            'type'    => ['bug'],
+            'issues'  => ['303'],
+            'authors' => ['pascal'],
+        ],
+        [
+            'message' => ['Gaia 1.2 (gaia_1_2) is no longer a supported repository source, replaced by Gaia 2.0 (gaia_2_0)'],
+            'section' => ['external_api'],
+            'type'    => ['change'],
+            'authors' => ['team'],
+        ],
+        [
+            'message' => ['New <code>start.sh</code> script at the root of the repository, launches dev mode, checks system dependencies and generates a working default config.ini file if it doesn’t exist, making getting started in development mode error free and even easier. Script entirey compatible with Mac/Linux (but not Windows yet)'],
+            'section' => ['developers'],
+            'type'    => ['new'],
+            'authors' => ['pascal'],
+        ],
+        [
+            'message' => ['<code>start.sh -remote</code> allows launching the webserver with 0.0.0.0 IP to make it accessible to the host OS if you develop in a VM'],
+            'section' => ['developers'],
+            'type'    => ['new'],
+        ],
+        [
+            'message' => ['<code>Utils::logScriptPerformances()</code> now returns correct rounded values'],
+            'section' => ['developers'],
+            'type'    => ['bug'],
+            'authors' => ['pascal'],
+        ],
+        [
+            'message' => ['Check if Composer (PHP dependency manager) is not installed globally before downloading it'],
+            'section' => ['developers'],
+            'type'    => ['better'],
+            'authors' => ['Julien Itard'],
+        ],
+        [
+            'message' => ['<code>Strings::endsWith()<code> and </code>Strings::startsWith()</code> now accept arrays of strings for checking, e.g. <code>Strings::startsWith(\'hello!\', [\'!\', \'?\', \'.\'])</code>'],
+            'section' => ['end_user'],
```

Comment:
Not sure if this one is causing the problem, but the first issues in 3.5 are missing the title.